### DISCUSSION TITLE
Odd look of Admin page

### DISCUSSION THREAD

**Discussion by kudlaty79bc** [netbox-community/netbox#6366] [Q&A] [answered]
Hello,
I installed Netbox from RHEL 8 repo and when I went to Admin page I found something like this:
![Admin_page](https://user-images.githubusercontent.com/45228759/117308431-d3103100-ae81-11eb-8251-e105b5a0cb2f.JPG)
Does anyone know what is wrong?

**Comment by candlerb** [comment, marked as answer]
It could be that you haven't configured your proxy (Nginx/Apache) for serving static assets; or the proxy doesn't have permissions to read the static assets; or you haven't run the `./upgrade.sh` script (which does the "collectstatic" step)

Being RHEL, it could also be SELinux which is preventing the proxy from reading the static assets.

See:
* https://github.com/netbox-community/netbox/tree/develop/contrib
* https://netbox.readthedocs.io/en/stable/installation/3-netbox/#run-the-upgrade-script

**Reply by kudlaty79bc** [reply]
Thank you for answer.
You were right, the configuration of static assets was incorrect.

For future reference:
When installing NetBox from repo on RHEL,  the correct path for /static is:
/var/lib/netbox/static

**Reply by jeremystretch** [reply]
The documentation uses `/opt/netbox` as the installation path. If you follow the installation guide consistently you won't run into this issue.

**Reply by kudlaty79bc** [reply]
Agree, but the documentation refers to the installation from the tar/zip file or by using git. When you install from RHEL OS repository the installation directories are completely different, thus you need to adjust information in the manual to your particular case.
